[IMP] pos/restaurant: Update Restaurant features page with child pages (#14048)
odoo/documentation: content/applications/sales/point_of_sale/restaurant.rst

===================
Restaurant features
===================

Odoo Point of Sale provides various features to manage a restaurant or a bar:

- :ref:`organizing the floors and tables <restaurant/floors>`;
- :ref:`taking orders <restaurant/orders>`;
- :ref:`communicating with the kitchen or bar through the POS <restaurant/orders-printing>`;
- :ref:`printing and splitting bills <restaurant/bills>`;
- :ref:`collecting tips <restaurant/tips>`;
- :doc:`setting different taxes for takeaway food <pricing/fiscal_position>`.

.. _restaurant/configuration:

Configuration
=============

To enable the restaurant and bar-specific features, follow these steps:

#. Go to :menuselection:`Point of Sale --> Configuration --> Settings`.
#. Select the Point of Sale in the top-left dropdown menu.

> #. Activate :guilabel:`Is a Bar/Restaurant` under the :guilabel:`Restaurant Mode` section.

Author: Yes, I agree, we created a task [https://www.odoo.com/odoo/project/3835/tasks/4929582] for it because we realised such page was missing.

   - The :icon:`fa-copy` (:guilabel:`Clone`) icon adds a table.
   - The :icon:`fa-trash` (:guilabel:`Delete`) icon removes the table.
#. Click :guilabel:`Save`.

.. warning::
   Removing a table or a floor is permanent.

.. _restaurant/floors/transfer:

Table transfer
--------------

To transfer an order to another table from the :guilabel:`Plan` view, follow these steps:

#. Select a table to go to the POS interface.
#. Start an order.
#. Click :menuselection:`Actions --> Transfer/Merge`.
#. In the :guilabel:`Plan` view, choose the target table:

   - Select a free table to transfer customers and their orders.
   - Select an occupied table to merge customers and their orders.

.. _restaurant/orders:

Order management
================

To take an order, open the register and follow these steps:

#. Select a :guilabel:`Plan` view.
#. Click a table to access the POS interface.
#. Start taking the order. The system automatically associates the order with the table.
#. Click :guilabel:`Order` to validate the order.
#. Click :guilabel:`Plan` to return to the :guilabel:`Plan` view.

To proceed with the order payment, click on the related table in the :guilabel:`Plan` view, then

> follow these steps:

Author: It would fit in the "Bills" section, indeed, but I included it as part of the order management because that's the expected flow when taking orders. Doing this would also avoid dragging readers across sections

#. Determine the product availability.
#. Click :guilabel:`Save & Close` or :guilabel:`Save & New`.

Kitchen preparation
-------------------

From an open POS session, start :ref:`taking an order <restaurant/orders>`, then click
:guilabel:`Order` to send it to the bar or the kitchen.

.. _restaurant/bills:

Bills
=====

There are two features to request or split a bill before payment: :ref:`Receipt printing
<restaurant/bills/printing>` and :ref:`Bill splitting <restaurant/bills/splitting>`.

Configuration
-------------

To allow bill printing and splitting, go to :menuselection:`Point of Sale --> Configuration -->
Settings`, select the POS, and enable the :guilabel:`Early Receipt Printing` and :guilabel:`Allow
Bill Splitting` features in the :guilabel:`Restaurant Mode` section.

.. _restaurant/bills/printing:

Receipt printing
----------------

From an open POS session, follow the :ref:`order process <restaurant/orders>` until the payment
is successful, then click :icon:`fa-print` :guilabel:`Print Full Receipt` to generate and print a
bill.

.. note::
   Any change to the order is reflected in the bill.

.. _restaurant/bills/splitting:

Bill splitting
--------------

> From an open POS session, start :ref:`selecting items <restaurant/orders>` and follow these steps:

Reviewer: Not sure what you mean here, I tested this in 18.0, no items selected; I was able to click the Actions and Split.
Also, I don't think you can select multiple items before you select the Split action anyways.

#. Click :menuselection:`Actions --> Split`.

> #. Select the items to split.

Reviewer: Sorry, I meant the wording wasn't really clear: items to split = e.g., a 1L bottle of water that several people share so you split the amount between several guests (which I'm not sure is possible in Odoo, is it?)

#. Click :guilabel:`Split Order`.
#. Proceed with the payment.
#. Click :icon:`fa-chevron-right` :guilabel:`Continue` and repeat the process for each guest.

.. note::
   The :icon:`fa-files-o` :guilabel:`Split` button is available when at least two items are ordered.

> 

Author: Yes, it is split into a "secondary" order: Order 4 when split gets order 4B. It can't go pass B, but you can split 4 again if there are multiple guests. I'll show you. And yes, we can add it to a note

.. _restaurant/tips:

Tips
====

> Point of Sale allows tipping in :ref:`shops <pos/sell>`, bars, and restaurants.

Author: I asked Mohamed (mobt) about it, he suggests keeping it in *Restaurant features* as it's widely spread in restaurant/bars, and quite rare in retail

#. (Optional) Tick the :guilabel:`Add tip after payment` and/or :guilabel:`Add tip through payment
   terminal (Adyen)` checkboxes for additional tipping configuration:

   - :guilabel:`Add tip after payment` generates a bill that indicates the tip value that the
     customer gives after the payment.
   - :guilabel:`Add tip through payment terminal (Adyen)` enables tipping with an :doc:`Adyen
     <../../../applications/sales/point_of_sale/payment_methods/terminals/adyen>` payment terminal.
#. Click :guilabel:`Save`.

.. important::
   - To use the :guilabel:`Add tip after payment` feature, the selected payment method must have a
     bank journal attributed to it.
   - To use the :guilabel:`Add tip through payment terminal (Adyen)` feature, select
     :guilabel:`Adyen` under the :guilabel:`Payment Terminals` section in the settings.

.. note::
   - When you create a product to use as a tip, leave the **product type** as :guilabel:`Consumable`
     to avoid unnecessary inventory movements.
   - You can only select one tip product per POS, but you can choose a different one for each.

.. _restaurant/tips/add-tips:

> Add tips during checkout

Reviewer: I don't think we would use the word "checkout" for a restaurant

     :ref:`payment terminal <terminals/configuration>`.
   - The :guilabel:`Add tip through payment terminal (Adyen)` setting only works with an
     :ref:`Adyen <adyen/tips>` terminal.

.. note::
   - Saving the :guilabel:`Tips` setting automatically fills the :guilabel:`Tip product` field
     with the preconfigured :guilabel:`[TIPS] Tips` product, which is only used for tips. When
     selecting another product in the :guilabel:`Tip product` field, the selected product no
     longer appears on the :ref:`POS interface <pos/restaurant/orders>`.
   - Choose only one tip product per POS.

.. _pos/restaurant/tips/add-tips:

Tip and payment
---------------

To process a tip during :ref:`payment <pos/restaurant/bills/payment>`, follow these steps:

#. On the :guilabel:`Payment` screen, select a :doc:`payment method
   <../../../applications/sales/point_of_sale/payment_methods>` for the order.
#. Click :icon:`fa-heart` :guilabel:`Tip`, add the tip amount in the popover, then click
   :guilabel:`Ok`.
#. Select a payment method for the tip and click :guilabel:`Validate`.

US tip setting
--------------

> If the :guilabel:`Add tip after payment` setting is enabled for a POS in the United States of
> America, follow these steps to process a tip:

Reviewer: I think I'd repeat the steps here, or at least the one about enabling the Add tip after payment setting?
```suggestion
To allow tipping after payment for a POS in the United States of
America, make sure the :guilabel:`Add tip after payment` setting is enabled in the POS settings. To process tips after payment, follow these steps:
```

#. On the :guilabel:`Payment` screen, select a :guilabel:`Card` payment method linked to a

>    :doc:`Stripe </../../../applications/sales/point_of_sale/payment_methods/terminals/stripe>` or
>    :doc:`Adyen </../../../applications/sales/point_of_sale/payment_methods/terminals/adyen>`
>    terminal.

Reviewer: ```suggestion
   :doc:`Stripe <payment_methods/terminals/stripe>` or :doc:`Adyen <payment_methods/terminals/adyen>`
```

> #. Click :guilabel:`Close Tab` to display the :guilabel:`Add a tip` screen with the following
>    options:

Reviewer: ```suggestion
#. Click :guilabel:`Close Tab` and select the relevant option in the :guilabel:`Add a tip` screen:
```

   - :guilabel:`15%`, :guilabel:`20%`, or :guilabel:`25%`: Tip rates based on order total.
   - :guilabel:`No Tip`.

>    - :guilabel:`Tip Amount`: Manual tip addition.

Reviewer: ```suggestion
   - :guilabel:`Tip Amount`: Enter the relevant amount in the field.
```
